lib/menu.dart

```diff
@@ -77,6 +78,14 @@ class Menu extends StatelessWidget {
         leading: const Icon(Icons.settings),
         onTap: () {},
       ),
+      ListTile(
+        title: const Text("Onboarding"),
+        leading: const Icon(Icons.help),
```

Comment:
after rebasing on `main`, you can follow the pattern to make it selectable

Reply:
Ah, crap, that's not supposed to be there, I committed that accidentally.

Reply:
I removed it in another fixup - if you think it's useful to keep it in the menu we can also keep it.

Reply:
I think it is useful indeed! if someone wants to trigger it, they can. makes the app feel a bit more complete.

Reply:
OK, I can add it in a separate commit and then we can decide :)

Reply:
> after rebasing on main, you can follow the pattern to make it selectable

This actually does not make much sense, because the onboarding tour is a separate set of screens that does not have a burger menu. I can set it to selected, but you would basically never see it 😅